### ISSUE TITLE
Support `urllib3>=2.0.0`

### ISSUE THREAD

**Issue by kripnerl** [boto/botocore#2951]
### Describe the feature

It would be nice if package requirements allow urlib3^2.0. 

The current requirements 'urllib3>=1.25.4,<1.27' is quite restrictive to use with newer projects. 

### Use Case

I am not able to use a package which requires boto3 together with a package which requires urlib3>2.0.

### Proposed Solution

'urllib3>=1.25.4,<2.0.2'

### Acknowledgements

- [ ] I may be able to implement this feature request
- [x] This feature might incur a breaking change

### SDK version used

latest

### Environment details (OS name and version, etc.)

any

**Comment by nateprewitt** [comment]
Hi @kripnerl, thanks for opening a ticket. This appears to be a duplicate of #2926. I'm going to resolve this in favor of the other as the primary tracking ticket. You can find a full description of the current state of things in [this comment](https://github.com/boto/botocore/issues/2926#issuecomment-1554767670).

While you're here though, I am curious what library or feature in urllib3 is prompting the upgrade requirement?

**Comment by kripnerl** [comment]
> 

Thank you for answerin!. I have overlooked  #2926... Not much familiar with GitHub search engine. 

Honesty, I don't know since I do not use  urlib3 directly. From what I understand, urlib 2.0.0 have some syntax improvements. 

I think I came across at least one project (https://github.com/hippalectryon-0/xr-scipy/) requiring the newer version. Since is rather a small project, I could create MR there and decrease its requirements; however, I found this rather back heading change.
